apps/standalone/src/app/hooks/useCliArtifacts.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { fetchCliArtifacts } from '../utils/apiCalls';
+
+export const useCliArtifacts = () => {
+  const getCliArtifacts = React.useCallback(
+    async <R>(abortSignal?: AbortSignal): Promise<R> => fetchCliArtifacts<R>(abortSignal),
```

Comment:
Why generic `R` ? We know the format of returned value

Reply:
True.
I moved the types to `extraTypes` and the function signature in  `AppContext` already uses it.